### DISCUSSION TITLE
Nomination of @J-N-K

### DISCUSSION THREAD

**Discussion by kaikreuzer** [openhab/openhab-addons#14711] [Add-on Maintainers Discussions]
Dear @openhab/add-ons-maintainers,

I'd like to nominate @J-N-K for again becoming a maintainer of this repo.
He is a long-term add-on contributor and a maintainer of many of these and I guess all of you are familiar with his valuable contributions, comments and general support.

Please therefore be so kind and cast your vote on this post by adding 👍 or 👎 .

Thanks!

**Comment by fabgio** [comment]
Congrats @J-N-K !

**Comment by kaikreuzer** [comment]
Please note: Only @openhab/add-ons-maintainers are allowed to vote - I see already many more 👍 than we have people in the maintainers team. 😆

**Reply by fabgio** [reply]
uh I apologize...

**Comment by kaikreuzer** [comment]
Ok, (more than) a week has passed without any maintainer's veto - so big welcome to @J-N-K!

**Comment by J-N-K** [comment]
Thanks!
